### ISSUE TITLE
Calendar ignores IsTodayHighlighted property

### ISSUE THREAD

**Issue by bradley-dotnet** [MahApps/MahApps.Metro#2332]
In version 1.2.4 (released) regardless of setting IsTodayHighlighted, today is highlighted. This makes it impossible to determine if it is currently selected. This code:

```
<Calendar SelectionMode="MultipleRange" DisplayDateStart="{x:Static sys:DateTime.Now}" 
                      IsTodayHighlighted="False"/>
```

will reproduce the issue. Under my color scheme (done using overrides) it looks like this:
![error](https://cloud.githubusercontent.com/assets/13209721/12692009/5a633f40-c6a6-11e5-8edd-eb3a7c16292a.png)

Both the 29th and the 30th are selected, but it looks like only the 30th is. I have yet to find a way to override this behavior. (I tried setting AccentColorBrush to Transparent, but because selection uses this brush as well, it breaks when selecting any dates).

**Comment by punker76** [comment]
@bradley-dotnet It's now better to see today as selected

![2016-03-26_22h35_11](https://cloud.githubusercontent.com/assets/658431/14062541/59a00488-f3a3-11e5-8ebb-db8d446ec09c.png)
![2016-03-26_22h35_19](https://cloud.githubusercontent.com/assets/658431/14062542/5b396f64-f3a3-11e5-9d1b-809a82665ff7.png)

**Comment by bradley-dotnet** [comment]
@punker76 That's great! It still looks like the code ignores the IsTodayHighlighted property though. Granted, this solves the "breakingness" of my original problem :)

**Comment by punker76** [comment]
@bradley-dotnet I'll look for the IsTodayHighlighted problem too

**Comment by bradley-dotnet** [comment]
Thanks, I'll post my workaround/solution tomorrow sometime in case it helps :)

**Comment by punker76** [comment]
@bradley-dotnet It's late, but never too late :-D This is fixed now and available in 1.5.0 (and pre-release)
